src/constants/ShoukakuConstants.js

```diff
@@ -70,13 +70,15 @@ class ShoukakuConstants {
     * @property {boolean} [noReplace=true] Specifies if the player will not replace the current track when executing this action.
     * @property {?number} [startTime] In milliseconds on when to start.
     * @property {?number} [endTime] In milliseconds on when to end.
+    * @property {boolean} [pause=false] If `true`, the player will pause when the track starts playing.
     * @memberof ShoukakuConstants#
     */
     static get ShoukakuPlayOptions() {
         return {
             noReplace: true,
             startTime: undefined,
-            endTime: undefined
+            endTime: undefined,
+            pause: false
```

Comment:
is the pause here optional on this new feature?

Reply:
yep, it's optional

Reply:
just move the pause below noReplace, then update the docs to reflect the changes